x-pack/solutions/security/plugins/security_solution/common/siem_migrations/types.ts

```diff
@@ -36,3 +36,5 @@ export type ItemDocument = RuleMigrationRule | DashboardMigrationDashboard;
 export type OriginalItem<I> = I extends RuleMigrationRule
   ? RuleMigrationRule['original_rule']
   : DashboardMigrationDashboard['original_dashboard'];
+
+export type MigrationType = 'rule' | 'dashboard';
```

Comment:
This is great, but we already have this type on the server side:
https://github.com/elastic/kibana/blob/7319260cece632687e5f1dd5e56a1d858c5e4bf5/x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/common/data/types.ts#L13

Would you mind removing this one on the _/server_, and always using the one at the _/common_? You'll only need to change it in `siem_migrations_data_item_client.ts`.